src/Components/WebAssembly/Server/src/DebugProxyLauncher.cs

```diff
@@ -43,7 +43,8 @@ private static string GetIgnoreProxyForLocalAddress()
         var noProxyEnvVar = Environment.GetEnvironmentVariable("NO_PROXY");
         if (noProxyEnvVar is not null)
         {
-            if (noProxyEnvVar.Equals("localhost") || noProxyEnvVar.Equals("127.0.0.1"))
+            var noProxyEnvVarValues = noProxyEnvVar.Split(",", StringSplitOptions.None);
+            if (noProxyEnvVarValues.Any(noProxyValue => noProxyValue.Equals("localhost") || noProxyValue.Equals("127.0.0.1")))
```

Comment:
Would it be worth using `StringSplitOptions.TrimEntries` so it's also allowed to have spaces in the string?

I notice your original example was `.example.com, .sub.example.com` but without trimming it wouldn't work to specify `.example.com, .sub.example.com, localhost`.